config/_shared.json

```diff
@@ -161,6 +161,7 @@
 		"jetpack-cloud-agency-signup": false,
 		"jetpack-cloud-auth": false,
 		"jetpack-cloud-partner-portal": false,
+		"jetpack-cloud-partner-program-signup": false,
```

Comment:
It's good practice to also set this to `false`in the other config files, just because they will be set to `true` there when we release.

For example: https://github.com/Automattic/wp-calypso/commit/7222c181b92b56170d077b711d3286f9afb9e957

Reply:
I considered that; the only reason I didn't is because they hadn't done that in the other reference PR. :^) But I'll add it anyway.

Reply:
e2ed373ec5